platform/broadcom/sonic-platform-modules-ruijie/common/script/device_i2c.py

```diff
@@ -0,0 +1,358 @@
+#!/usr/bin/env python
```

Comment:
Python 2 is deprecated and no longer supported. We are in the process of migrating all Python code in SONiC to Python 3. Please change this shebang to `#!/usr/bin/env python3` and ensure the script is Python 3-compliant.